CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+# Contribution to Enmeshed
+
+You can contribute to Enmeshed with issues and PRs. Simply filing issues for problems you encounter is a great way to contribute. Contributing implementations is greatly appreciated.
```

Comment:
PRs -> PullRequests

Reply:
"Contributing implementations" -> "Contributing bug fixes or features according to already filed bugs or feature requests (see feedback repository) is greatly appreciated".

Reply:
Ich störe mich einfach an dem "contribution implementations" - was genau sind "implementations"? :)

Reply:
Jo, versteh ich. Ich nehm deine Implementierung und passe sie noch ein wenig an